WORKSPACE

```diff
@@ -101,8 +101,10 @@ install_bazel_dependencies()
 # With http_archive it only sees releases/download/*.tar.gz urls
 git_repository(
     name = "build_bazel_rules_typescript",
-    commit = "5d79e42953eb8614d961ccf0e3440884e974eeb3",
+    commit = "9f4e2da7ea5ac2eb992b897f2a63cea66732fb62",
+    patches = ["//:rules_typescript.patch"],
```

Comment:
patch may no longer be needed now that the underlying BAZEL_NODE_RUNFILES_HELPER issue is fiexed

Reply:
nope, still needed, I introduced a logic bug only observable when using workers outside of the local Runfiles case (like in `@bazel/worker`)